node-deb-executable

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
This file is a bit redundant. I think a better change here would be to append the `$@` into the default executable so it's the default for everyone. Or just in our case, it could be added to the `package.json` as the line `"start": "./node-deb $@"`. I think the former makes the most sense.